### PR TITLE
Traversal of the space of train test splits, fixes #3

### LINKED ISSUE #3: [Outreachy applications] Traversal of the space of train/test splits
Given a classification model, we want to investigate how much the performance score computed on the test set depends on the choice of train/test split proportion. Eg. how would our performance estimate change if we used a 60/40 split rather than 80/20?

Write a function that takes a scikit-learn estimator and a dataset, and computes an evaluation metric over a grid of train/test split proportions from 0 to 100%. To assess variability, for each split proportion it should resplit and recompute the metric multiple times. It should output a table of splits with multiple metric values per split.

### PR DESCRIPTION
**Pull request by Sidrah-Madiha**
This branch attempts to fixes #3, in this branch there is a folder > "Traversal_of_the_space_of_train_test splits", 
- this folder contains a helper function file "helper_traversal_train_test_splits.py", this function splits  test data from 1-99 percent range with step of 5 percent  (I am seeing issue on split 0 or 100  and step of 1 percent - will fix in next iteration) and for each such split, it takes training data and records cross validation score for 2-5 folds, the output is a tabular data as shown in fig:
train1 , 
- also for each split , a visualisation of score for each fold is displayed, for each split a separate plot is drawn (please see image below), in next iteration I will fix it so that all plots appear side by side for comparison Or may be I will scratch that visualisation and plot is some other way. 
train2

### CODE REVIEW

**Comment by Sidrah-Madiha** [comment]
The updated pull request https://github.com/mozilla/PRESC/pull/13 for start up task # 2, fixes #3 as well.
Please see train_test_split_v2.py file, it has 2 function, one function returns a table shown below: 
Splits
the next function is implemented for visualising mean test score and train scores for a given training size, as shown in the attached image: 
LC, 
I think in my latest pull request https://github.com/mozilla/PRESC/pull/13, I have implemented much better solution of train/test split then the previous PR https://github.com/mozilla/PRESC/pull/46, I will probably close pull request 46.
@dzeber  please suggest which of these PRs (https://github.com/mozilla/PRESC/pull/13, https://github.com/mozilla/PRESC/pull/46)  has a better implementation for train/test split:

**Comment by mlopatka** [comment]
@Sidrah-Madiha Can you please rebase against master?
If you need help, [this](https://stackoverflow.com/questions/7929369/how-to-rebase-local-branch-with-remote-master) is a useful thread.

Thee cause of this is that you have many PRs in flight that use the same files that we have not been able to keep up with reviewing fast enough. So when I merge in an older version of your code from a separate PR there is a conflict between the (now master) version of your code and the outstanding PR version of your code. 

Fortunately, since all commits are local to your own dev/ part of the repo, the rebase should go smoothly. 

Please flag me here for review and merge when the conflicts are all resolved.
Sorry again for the delay in getting to these reviews.

**Comment by Sidrah-Madiha** [comment]
@mlopatka I tried fixing the issue with this solution https://stackoverflow.com/questions/56683681/this-branch-has-conflicts-that-must-be-resolved-but-its-already-merged after rebasing, the branch now contains files that are present in master, but I am still seeing "This branch has conflicts that must be resolved" in this PR, can you please advise me how to proceed to resolve this conflict?

**Comment by Sidrah-Madiha** [comment]
@mlopatka finally fixed the merge conflict. The issue was that in my master on mozilla PRESC (https://github.com/mozilla/PRESC/tree/master/dev/Sidrah-Madiha) there are 2 files missing that are present on my forked master (https://github.com/Sidrah-Madiha/PRESC/tree/master/dev/Sidrah-Madiha), I was worried that my PR solution (https://github.com/mozilla/PRESC/pull/13) will not be visible due to this discrepancy so I have added a folder (Traversal_of_the_space_of_train_test_splits/Another_version_of_fix_of_3 ) which shows my PR 13 in its entirety now which also has a different and efficient approach for fixing #3 Please review
